eunkyung/controllers/likeController.js

```diff
@@ -0,0 +1,11 @@
+const likeService = require('../services/likeService')
+
+
+const likePost = async (req, res) => {
+    const { userId, postId } = req.body
+    await likeService.likePost(userId, postId)
+    return res.status(201).json({ message: 'I like this POST' })
```

Comment:
응답 메시지는 클라이언트, 즉 프론트엔드 개발자와 나중에 백엔드 개발자가 디버깅을 위한 로그를 확인할 때 볼 수 있는 에러메시지입니다.
조금 더 명확한 메시지를 작성해주면 좋을 것 같습니다!
```suggestion
    return res.status(201).json({ message: 'Like Success' })
```